src/app/search/search.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { Constants } from '../core/constant';
+
+@Component({
+  selector: 'app-search',
+  templateUrl: './search.component.html',
+  styleUrls: ['./search.component.css']
+})
+export class SearchComponent implements OnInit {
+  nextPage = Constants.transactions.nextPage;
+  constructor(private router: Router) { }
+
+  ngOnInit() {
+  }
+
+  onNextPage() {
+    this.router.navigate(['/transactional']);
```

Comment:
use constant variable here

Reply:
const variable has been used in HTML. In this method that variable is not needed